openedx/features/ucsd_features/ecommerce/EcommerceClient.py

```diff
@@ -10,10 +10,34 @@
 
 
 class EcommerceRestAPIClient:
+    """
```

Comment:
I think we normally use a snake case for the file name.

Reply:
Good catch. Updated file name